common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -69,6 +74,15 @@ class AccountInfo extends React.Component<Props, State> {
     });
   };
 
+  public onCopy = () => {
+    this.setState(state => {
+      return {
+        copied: !state.copied,
+        currentColor: 'blue'
```

Comment:
It would be preferable if style related stuff was handled in CSS, using our sass variables instead of introducing new colors. Can you have a different class added based on `copied`'s state, and style it in `AccountInfo.scss`?